sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -1871,12 +1871,15 @@ def clip(self, axis, **kwargs):
     """lower and upper must be :class:`DeferredSeries` instances, or constants.
```

Comment:
Are `lower` and `upper` arg names? I'm wondering if they should have backticks around them.

Reply:
Done, see comment above